src/main/java/uk/gov/hmcts/reform/civil/handler/callback/camunda/notification/ClaimContinuingOnlineApplicantPartyForSpecNotificationHandler.java

```diff
@@ -71,7 +71,7 @@ RESPONSE_DEADLINE, formatLocalDate(caseData.getRespondent1ResponseDeadline()
 
     private void generateEmail(CaseData caseData) {
         notificationService.sendMail(
-            caseData.getApplicant1().getPartyEmail(),
+            caseData.getClaimantUserDetails().getEmail(),
```

Comment:
perhaps we need a method getApplicantEmail and chose if there is no email in applicant1 party then we choose the email they logged in with. in this way if they do introduce the email field in cui we wont have to change the code again

Reply:
Done!